website/components/navigation/MobileNavigation.tsx

```diff
@@ -109,7 +109,7 @@ function MobileNavigation({ children }: { children: React.ReactNode }) {
           >
             <MenuIcon size={20} />
           </IconButton>
-          <MDBDesignLogo />
+          <MDBDesignLogo onClick={() => push('/')} />
```

Comment:
can we store a constant somewhere for the path, e.g. `const HOME_PAGE = '/';`

Reply:
yep